### PR TITLE
Update dependency list in the README

### PR DESCRIPTION
**Pull request by aduh95**
This PR aims to update the list of dependencies in the README file.

I couldn't find an explanation for every dependencies, I thought we could discuss this in this PR - or maybe ditch the entire section of the README as it is going to be a pain in the neck to maintain...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,13 +106,17 @@ You can run the tests with `cargo test`.
...
 ## Built With
 
-External crates :
+External crates:
 
-- `curl` for sending HTTP requests.
-- `serde_json` for JSON parsing.
-- `percent-encoding` to encode search queries in percent encoding.
-- `termion` and `tui` for the text user interface
+- `base64`
```
Comment (lethenju) [inline README.md]:
```suggestion
- `base64` for encoding the application credentials pair.
```

File: README.md [hunk 2]
```diff
@@ -106,13 +106,17 @@ You can run the tests with `cargo test`.
...
-External crates :
+External crates:
 
-- `curl` for sending HTTP requests.
-- `serde_json` for JSON parsing.
-- `percent-encoding` to encode search queries in percent encoding.
-- `termion` and `tui` for the text user interface
+- `base64` 
 - `clap` for the command line interpreter
+- `failure`
```
Comment (lethenju) [inline README.md]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -106,13 +106,17 @@ You can run the tests with `cargo test`.
...
+- `tiny_http` for the automated refresh token generation process
+- `term-painter`
```
Comment (lethenju) [inline README.md]:
```suggestion
- `term-painter` for colorful stdout during the automated refresh token generation process
```

File: README.md [hunk 4]
```diff
@@ -106,13 +106,17 @@ You can run the tests with `cargo test`.
...
+- `termion` for the text user interface
+- `text_io`
```
Comment (lethenju) [inline README.md]:
```suggestion
- `text_io` for easy user input 
```

File: README.md [hunk 5]
```diff
@@ -106,13 +106,17 @@ You can run the tests with `cargo test`.
...
+- `tui` for the text user interface
+- `webbrowser`
```
Comment (lethenju) [inline README.md]:
```suggestion
- `webbrowser` for the automated refresh token generation process
```

File: README.md [hunk 6]
```diff
@@ -106,13 +106,17 @@ You can run the tests with `cargo test`.
 
 ## Built With
 
-External crates :
+External crates:
 
-- `curl` for sending HTTP requests.
```
Comment (lethenju) [inline README.md]: why do you want to delete those lines (111, 112, 113) ?

**Review by lethenju — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (aduh95) [reply README.md]: Isn't that in the `Cargo.toml`?

https://github.com/lethenju/spotify_rust/blob/635850eb57ef81001400715a26383f521793dc46/Cargo.toml#L11

File: README.md (hunk 6 above)
Comment (aduh95) [reply README.md]: It's not an external crate, is it?

Comment (aduh95) [reply README.md]: Oh in this `Cargo.toml`: https://github.com/lethenju/spotify_rust/blob/master/spotify_api/Cargo.toml

Let me fix that then!

File: README.md (hunk 2 above)
Comment (aduh95) [reply README.md]: What about that?

```suggestion
- `failure` for error handling
```

**Review by lethenju — APPROVED** [review]

**Pull request merged**